.coveragerc

```diff
@@ -9,8 +9,9 @@ exclude_lines =
     raise NotImplementedError
 
 ignore_errors = True
-fail_under = 80
+fail_under = 65
```

Comment:
We should update this to something higher before we merge to master. Ideally 80%, but IDK if we will get test coverage that high.